core/commands/dns.go

```diff
@@ -26,7 +26,7 @@ This command resolves those links to the referenced object.
 
 For example, with this DNS TXT record:
 
-	> dig +short TXT ipfs.io
+	> dig +short TXT _dnslink.ipfs.io
```

Comment:
this change does not need to happen. it can be either. I'm not sure we _want_ it to happen.

the documentation reads "with this X, then Y" it does not require a working DNS record, and it could use `example.com` instead.